src/Model/Table/AclNodesTable.php

```diff
@@ -154,7 +154,7 @@ public function node($ref = null)
             foreach ($ref as $key => $val) {
                 if (strpos($key, $type) !== 0 && strpos($key, '.') === false) {
                     unset($ref[$key]);
-                    $ref["{$type}0.{$key}"] = $val;
+                    $ref[$type . '0.' . $key . ((is_null($val)) ? ' IS' : '')] = $val;
```

Comment:
You can simply do `"{$type}0.{$key} IS"` as the ORM will automatically remove the `IS` if the value is not `null`